README.md

```diff
@@ -217,7 +217,7 @@ If you use [Carthage][] to manage your dependencies, simply add
 DrawerKit to your `Cartfile`:
 
 ```
-github "DrawerKit/DrawerKit" ~> 0.0.1
+github "DrawerKit/DrawerKit" ~> 0.1.1
```

Comment:
We shouldn't lock on a patch level, this should either lock on `0.1` or `1.0` with my preference for the later.

Reply:
Fixed in https://github.com/Babylonpartners/DrawerKit/pull/21/commits/76e2ce3b2c5cb3ad7de5372b9191c080ffd4b67d.